app-text/notes/notes-2.3.1.recipe

```diff
@@ -53,6 +53,8 @@ REQUIRES="
 	lib:libQt6Network$secondaryArchSuffix
 	lib:libQt6Sql$secondaryArchSuffix
 	lib:libQt6Widgets$secondaryArchSuffix
+        lib:libQt6Qml$secondaryArchSuffix
+        lib:libQt6Quick$secondaryArchSuffix
```

Comment:
This should come between Qt6Network and Qt6Sql, also the indentation is wrong (use a single tab).